NHSN/src/test/java/com/nhsn/testcases/TC06_VerifySubmitDateOnReviewPage.java

```diff
@@ -0,0 +1,42 @@
+package com.nhsn.testcases;
+
+import com.nhsn.BaseClass;
+import com.nhsn.helpers.CommonFunctions;
+import com.nhsn.pages.HomePage;
+import com.nhsn.pages.LoginPage;
+import com.nhsn.pages.ReviewPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TC06_VerifySubmitDateOnReviewPage  extends BaseClass {
```

Comment:
Please start code-commenting these classes with an overview of what the class tests. i.e.
```
/**
   * 1. Opens the home page and logs in
   * 2. Navigates to the "Review" screen
   * 3. Specifies a "submission date" as filter criteria
   * 4. ...???
public class TC06_VerifySubmitDateOnReviewPage extends BaseClass {
```
Please make this standard practice across all of these classes. If there is more than one `@Test` in a given class, please describe what the tests do differently from the more generic description of the class.

Reply:
I updated the comments like above for all other test cases